### PR TITLE
Handle JSON array of releases

### PR DESCRIPTION
**Pull request by aguilerapy**
open-contracting#19

### CODE REVIEW

**Comment by aguilerapy** [comment]
open-contracting#19

**Comment by aguilerapy** [comment]
open-contracting#19

**Comment by jpmckinney** [comment]
@aguilerapy Can you add a test for the new behavior?

File: default/templates/default/package-releases.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
 in the box below to create a <a href="https://standard.open-contracting.org/latest/en/schema/release_package/" target="_blank">Release Package</a> {% endblocktrans %}.
 {% blocktrans %}You can also click the <strong>Add a file</strong> link below.{% endblocktrans %}
 {% blocktrans %}Use the <strong>Published Date</strong> option to set the Published Date in the output package.{% endblocktrans %}</p>
-<p>{% blocktrans %}The input files must be JSON files with a single Release each.{% endblocktrans%}</p>
+<p>{% blocktrans %}The input files must be JSON files of Releases and can be JSON Array files of Releases.{% endblocktrans%}</p>
```
Comment (romifz) [inline default/templates/default/package-releases.html]: Can you please change this to "or a JSON Array of Releases"?

Comment (jpmckinney) [reply default/templates/default/package-releases.html]: Please use sentence case, e.g. "or a JSON array of releases".

**Pull request merged**